tests/plugins/test_delete_from_registry.py

```diff
@@ -39,14 +39,14 @@
 
 
 class Y(object):
-    pass
+    def __init__(self):
+        self.dockerfile_path = None
+        self.path = None
```

Comment:
Alternatively, we could use `stubs.StubSource` instead of `Y`, and `stubs.StubInsideBuilder` instead of `X`.